### DISCUSSION TITLE
Upgrade issue 1.11 -> 1.13

### DISCUSSION THREAD

**Discussion by qualifiedink** [kimai/kimai#2412] [Questions & Answers]
Hi there:-

Today I tried to upgrade my installation from 1.11 to 1.13 following these instructions: https://www.kimai.org/documentation/updates.html. I could see any errors during the installation process. However, on my browser window I can read:

**Something is wrong**
A critical error occurred, please try again. You may have found a software problem, please contact your administrator if the problem persists.
<DASHBOARD>

Where can I find what went wrong and how can I fix it? Thx.

Best,
Markus

**Comment by kevinpapst** [comment]
Check your logs. Check and set file permissions (this is the problem in 99% of all cases)

**Reply by qualifiedink** [reply]
I thought I had this done. Running 

chown -R :www-data .
chmod -R g+r .
chmod -R g+rw var/
chmod -R g+rw public/avatars/

again did the trick. Thx!
